scheduler/config-composite.edn

```diff
@@ -61,7 +61,22 @@
            :user-metrics-interval-seconds 60}
  :nrepl {:enabled? true
          :port #config/env-int "COOK_NREPL_PORT"}
- :pools {:default "mesos-gamma"}
+ :pools {:default "mesos-gamma"
+         :job-resource-adjustment {:adjust-job-resources-fn cook.kubernetes.api/adjust-job-resources
+                                   :pool-regex "^k8s-.+"}
+         :default-container-for-pool
+         [{:pool-regex "k8s-alpha" :container {:type "docker"
+                                               :docker {
+                                                        :image "python:3.6",
+                                                        :network "HOST"
+                                                        :force-pull-image false
+                                                        :parameters []}}}
+          {:pool-regex "k8s-gamma" :container {:type "docker"
+                                               :docker {
+                                                        :image "python:3.6",
+                                                        :network "HOST"
+                                                        :force-pull-image false
+                                                        :parameters []}}}]}
```

Comment:
Shouldn't we combine these into one entry?

Reply:
Merged